hcn/hcnpolicy.go

```diff
@@ -205,3 +206,11 @@ type VlanPolicySetting struct {
 type VsidPolicySetting struct {
 	IsolationId uint32 `json:","`
 }
+
+// RemoteSubnetRoutePolicySetting creates remote subnet route rules on a network
+type RemoteSubnetRoutePolicySetting struct {
+	DestinationPrefix           string `json:","`
```

Comment:
You dont need `json:","` if you aren't overriding anything.

Reply:
True. It was done more for consistency.

Reply:
Did it for this type.  Didn't go back and do it for all others.